From e057d449be13eac5aef6c2fff985f9ebe12252d9 Mon Sep 17 00:00:00 2001
From: Kuninori Morimoto <kuninori.morimoto.gx@renesas.com>
Date: Wed, 15 May 2013 16:58:24 +0900
Subject: [PATCH 512/675] LTSI: ARM: shmobile: bockw: fixup early timer dtb
 compile

LTSI v3.4 ARM kernel doesn't support .init_time on DT_MACHINE_START,
and, doesn't have arch/arm/boot/dts/Makefile.
but, upstream base r8a7778 SoC support is using these.
This patch fixup upstream <--> LTSI v3.4 generation conflict

Signed-off-by: Kuninori Morimoto <kuninori.morimoto.gx@renesas.com>
---
 arch/arm/mach-shmobile/Makefile.boot | 1 +
 arch/arm/mach-shmobile/board-bockw.c | 2 +-
 2 files changed, 2 insertions(+), 1 deletion(-)

diff --git a/arch/arm/mach-shmobile/Makefile.boot b/arch/arm/mach-shmobile/Makefile.boot
index 5e41019..f353b30 100644
--- a/arch/arm/mach-shmobile/Makefile.boot
+++ b/arch/arm/mach-shmobile/Makefile.boot
@@ -11,3 +11,4 @@ __ZRELADDR	:= $(shell /bin/bash -c 'printf "0x%08x" \
 dtb-$(CONFIG_MACH_KZM9G) += sh73a0-kzm9g.dtb
 dtb-$(CONFIG_MACH_KZM9D) += emev2-kzm9d.dtb
 dtb-$(CONFIG_MACH_ARMADILLO800EVA) += r8a7740-armadillo800eva.dtb
+dtb-$(CONFIG_MACH_BOCKW) += r8a7778-bockw.dtb
diff --git a/arch/arm/mach-shmobile/board-bockw.c b/arch/arm/mach-shmobile/board-bockw.c
index 38e5e50..c58338a 100644
--- a/arch/arm/mach-shmobile/board-bockw.c
+++ b/arch/arm/mach-shmobile/board-bockw.c
@@ -76,6 +76,6 @@ DT_MACHINE_START(BOCKW_DT, "bockw")
 	.init_early	= r8a7778_init_delay,
 	.init_irq	= r8a7778_init_irq_dt,
 	.init_machine	= bockw_init,
-	.init_time	= shmobile_timer_init,
+	.timer		= &shmobile_timer,
 	.dt_compat	= bockw_boards_compat_dt,
 MACHINE_END
-- 
1.8.2

